test-utils/src/main/java/apoc/util/TestContainerUtil.java

```diff
@@ -272,7 +272,7 @@ public static void checkLeadershipBalanced(Session session) {
                         return false;
                     }
                 },
-                (value) -> value, 30L, TimeUnit.SECONDS);
+                (value) -> value, 60L, TimeUnit.SECONDS);
```

Comment:
This was needed because there seem to be a few flaky tests around `UUIDClusterRoutingTest`